notes/interview-summary.vbs

```diff
@@ -1,3 +1,6594 @@
+                    If IsNumeric(left(disa_verif_memb, 2)) = TRUE Then
+                        verifs_selected = verifs_selected & "Ill/Incap or Disability for Memb " & disa_verif_memb & " of " & disa_verif_type & ",; "
+                    Else
+                        verifs_selected = verifs_selected & "Ill/Incap or Disability for " & disa_verif_memb & " of " & disa_verif_type & ",; "
+                    End If
+                    illness_disability_checkbox = unchecked
+                    disa_verif_memb = ""
+                    disa_verif_type = ""
+                End If
+                other_verifs = trim(other_verifs)
+                If other_verifs <> "" Then verifs_selected = verifs_selected & other_verifs & "; "
+                other_verifs = ""
+            Else
+                MsgBox "Additional detail about verifications to note is needed:" & vbNewLine & verif_err_msg
+            End If
+
+			If verif_err_msg = "" Then
+				If ButtonPressed = add_verif_button Then verif_view = "Add A Verif"
+				If ButtonPressed = fill_button Then verif_view = "See All Verifs"
+			End If
+
+			If ButtonPressed = add_verif_button Then verif_err_msg = "LOOP" & verif_err_msg
+            If ButtonPressed = fill_button Then verif_err_msg = "LOOP" & verif_err_msg
+        Loop until verif_err_msg = ""
+        ButtonPressed = verif_button
+    End If
+
+end function
+
+function write_interview_CASE_NOTE()
+	' 'Now we case note!
+	STATS_manualtime = STATS_manualtime + 600
+	Call start_a_blank_case_note
+
+	CALL write_variable_in_CASE_NOTE("~ Interview Completed on " & interview_date & " ~")
+    Call write_bullet_and_variable_in_CASE_NOTE("Case Information", case_summary)
+    If cash_request = True and the_process_for_cash = "Application" and type_of_cash = "Family" Then
+        Call write_variable_in_CASE_NOTE("Family Cash Program Selection Details")
+        CALL write_bullet_and_variable_in_CASE_NOTE("Program selected", family_cash_program)
+        CALL write_bullet_and_variable_in_CASE_NOTE("Selection Notes", famliy_cash_notes)
+    End If
+
+	CALL write_variable_in_CASE_NOTE("Completed with " & who_are_we_completing_the_interview_with & " via " & how_are_we_completing_the_interview)
+	If trim(interpreter_information) <> "" AND interpreter_information <> "No Interpreter Used" Then
+		CALL write_variable_in_CASE_NOTE("Interview had interpreter: " & interpreter_information)
+		CALL write_variable_in_CASE_NOTE("    Language: " & interpreter_language)
+	End If
+	If trim(arep_interview_id_information) <> "" Then CALL write_variable_in_CASE_NOTE("AREP Identity Verification: " & arep_interview_id_information)
+	If trim(non_applicant_interview_info) <> "" Then CALL write_variable_in_CASE_NOTE("Interviewee Information: " & non_applicant_interview_info)
+	CALL write_variable_in_CASE_NOTE("Completed on " & interview_date & " at " & interview_started_time & " (" & interview_time & " min)")
```

Comment:
Guessing this time will pull in once it's hooked up to the interview script?

Reply:
No, good catch. We only want the date here as we don't have the time specific information for this run.